DEPS

```diff
@@ -15,7 +15,7 @@ deps = {
     "url": "https://github.com/brave/Sparkle.git@8721f93f694244f9ff41fe975a92617ac5f63f9a",
     "condition": "checkout_mac",
   },
-  "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@800f9d40b7409239ff192e0be634764e747c7a75",
+  "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@8b424ccf29957fe01c88c36076fd32006a357887",
```

Comment:
https://github.com/brave-intl/bat-native-tweetnacl/commit/8b424ccf29957fe01c88c36076fd32006a357887

Reply:
We should split the bump out into a separate PR? Or was it required here?